packages/google-cloud-cloudquotas/.OwlBot.yaml

```
@@ -0,0 +1,18 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+deep-copy-regex:
+    - source: /google/cloud/cloudquotas/(v.*)/.*-py
```

```suggestion
    - source: /google/api/cloudquotas/(v.*)/.*-py
```

To match the path in `googleapis-gen`
https://github.com/googleapis/googleapis-gen/tree/master/google/api/cloudquotas

---

Correct. I think the fix is in #12181 PTAL.